SPEC.md

```diff
@@ -109,13 +109,15 @@ A network configuration consists of a JSON object with the following keys:
 
 - `cniVersion` (string): [Semantic Version 2.0](https://semver.org) of CNI specification to which this configuration list and all the individual configurations conform. Currently "1.1.0"
 - `cniVersions` (string list): List of all CNI versions which this configuration supports. See [version selection](#version-selection) below.
-- `name` (string): Network name. This should be unique across all network configurations on a host (or other administrative domain).  Must start with an alphanumeric character, optionally followed by any combination of one or more alphanumeric characters, underscore, dot (.) or hyphen (-).
+- `name` (string): Network name. This should be unique across all network configurations on a host (or other administrative domain).  Must start with an alphanumeric character, optionally followed by any combination of one or more alphanumeric characters, underscore, dot (.) or hyphen (-). Must not contain characters disallowed in file paths. A path segment (such as a filesystem directory) with the same name as the network name, containing one or more plugin configuration JSON objects for that network, should exist at the same level as the network configuration object itself.
```

Comment:
I would drop the last sentence; this portion of the spec does not really concern configuration loading and parsing.

That the network name must be a valid file component is a useful (and implicit) restriction, so that's a good idea.

Reply:
> I would drop the last sentence; this portion of the spec does not really concern configuration loading and parsing.

Fair enough.

> That the network name must be a valid file component is a useful (and implicit) restriction, so that's a good idea.

Yeah, the intent was to explain _why_ the network name must be a valid filename, since saying it should and not explaining why feels a bit weird. Doing so does necessarily make config loading and parsing requirements more explicit/opinionated, tho.

Reply:
Removed the sentence, ty!